app/views/multiworlds/new.html.erb

```diff
@@ -0,0 +1,89 @@
+<br />
+<h1>Create Game</h1>
+<%= form_with url: '/multiworlds', method: :post do |form| %>
+  <div id="player-count-container">
+    <div class="row">
+      <div class="col-3">
+        <%= form.label :player_count %>
+        <br />
+        <%= form.number_field :player_count, min: 1, class: "form-control" %>
+      </div>
+    </div>
+    <br />
+    <div class="row">
+      <div class="col-3">
+        <button id="player-count-next" type="button" class="btn btn-primary">Next</button>
+      </div>
+    </div>
+  </div>
+  <div id="players-container" class="d-none row">
+    <div id="players">
+    </div>
+    <div id="select-players">
+    </div>
+    <div id="submit-game" class="col-3 d-none">
+      <%= form.submit "Create Game", class: "btn btn-primary" %>
+    </div>
+  </div>
+<% end %>
+
+<script type="text/javascript">
+  var player_template = handlebars.compile(`<%= render 'player' %>`)
+  var select_player_template = handlebars.compile(`<%= render 'select_player' %>`)
+
+  $('#player-count-next').on('click', function(event) {
+    const player_count = _.toInteger($('#player_count').val())
+    if (player_count < 2) {
+      alert('You must have at least 2 players to continue')
+    } else {
+      $('#player-count').text(player_count)
+
+      add_player({
+        number: 1,
+        name: '<%= current_user.username %>',
+        player_id: <%= current_user.id %>
+      })
+
+      _.times(player_count-1, function(index) {
+        add_select_player({number: index+2})
+      })
+      $('.player-select-button').on('click', submit_player)
+
+      $('#player-count-container').hide()
+      $('#players-container').removeClass('d-none')
+    }
+  })
+
+  function submit_player(event) {
+    player_number = $(event.target).data('player-number')
+    player_select = $('#player-select-'+player_number)
+    player_name = player_select.children(':selected').first().text()
+    player_id = player_select.val()
+    if (player_id == '') {
```

Comment:
I think you can just do `if (!player_id)`.

Also not sure if you want to juggle naming conventions but js has camel case, I think?